DESCRIPTION

```diff
@@ -17,8 +17,7 @@ Imports:
     devtools,
     httr
 Remotes:
-    annecori/EpiEstim@new-version,
-    nickreich/coarseDataTools,
```

Comment:
You might want to specify a minimum version for EpiEstim and coarseDataTools eg:

```
    coarseDataTools (>= 0.6-4),
    EpiEstim (>= 2.2-0)
```

Reply:
(not in the remotes section, though)